http_handlers.go

```diff
@@ -209,3 +210,56 @@ func getIPFromRequest(r *http.Request) string {
 	}
 	return IPAddress
 }
+
+func writeJson(w http.ResponseWriter, status int, data interface{}) error {
+	var body []byte
+	body, err := json.Marshal(data)
+	if err != nil {
+		if status < 400 {
+			return jsonError(w, http.StatusInternalServerError, err, "Marshalling response")
+		} else {
+			logger.Error(err, "Failed to marshal error")
+			body = []byte(`{"error", {"comment": "Failed to marshal error"}}`)
```

Comment:
not a fan of error messages in body, its a bit annoying if you ask me. Do we need/actually use it?

Reply:
no, but it helps with troubleshooting an issue. This particular error is a catch all.